### PR TITLE
GAWB-3600: Add new api call

### PR DESCRIPTION
**Pull request by rushtong**
See https://github.com/broadinstitute/firecloud-orchestration/pull/626
Supports https://broadinstitute.atlassian.net/browse/GAWB-3600

This is a new API call so no need for a version bump.

**PR checklist**
- [x] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [x] Bump the version in `project/Settings.scala` `createVersion()`
- [x] Update `CHANGELOG.md` for those libraries
- [x] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [x] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [x] Get two thumbsworth of PR review
- [x] Verify all tests go green (CI _and_ coverage tests)
- [x] Squash commits and **merge to develop**
- [x] Delete branch after merge

### CODE REVIEW

**Comment by rushtong** [comment]
@MatthewBemis and @breilly2 - If you have a few extra minutes, I'd like to know if I've followed the instructions correctly for this PR. Thank you!

**Review by breilly2 — APPROVED** [review]
👍 LGTM

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ To depend on the `MockGoogle*` classes, additionally depend on:
 
 Contains common classes and utilities for writing tests against Workbench REST web services.
 
-Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-service-test" % "0.9-2072b3e" % "test" classifier "tests"`
+Latest SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-service-test" % "TRAVIS-REPLACE-ME" % "test" classifier "tests"`
```
Comment (MatthewBemis) [inline README.md]: travis only fills in the git hash, not the version number, so you'll want to make sure this is `0.9-TRAVIS-REPLACE-ME`

(same for CHANGELOG.md)

**Review by MatthewBemis — APPROVED** [review]
1 comment, then :+1: after fixing

File: README.md (hunk 1 above)
Comment (rushtong) [reply README.md]: Thank you!!!

**Pull request merged**